src/SprykerEco/Zed/Heidelpay/Business/Adapter/Payment/DirectDebitPayment.php

```diff
@@ -57,34 +57,4 @@ public function refund(HeidelpayRequestTransfer $refundRequestTransfer)
 
         return $this->verifyAndParseResponse($directDebitPaymentMethod->getResponse());
     }
-
-    /**
-     * @param \Generated\Shared\Transfer\HeidelpayRequestTransfer $authorizeRequestTransfer
-     *
-     * @return \Generated\Shared\Transfer\HeidelpayResponseTransfer
-     */
-    public function authorize(HeidelpayRequestTransfer $authorizeRequestTransfer): HeidelpayResponseTransfer
```

Comment:
Why this is removed?

Reply:
Authorization is not part of our scope. We are going to have implementation based immediate debit after registration.